.github/workflows/backport-has-issue.yml

```diff
@@ -0,0 +1,21 @@
+name: Backport has Issue
+on:
+  pull_request_target:
+    on:
+      push:
+        branches:
```

Comment:
This does not seem right. Have you tested this?

AFAIK, `branches` should be directly under `pull_request_target`.

Reply:
Fixed!

Reply:
Fixed!